src/index.html

```diff
@@ -10,8 +10,219 @@
+            Full date of birth:
+            <input
+            type="date"
+            name="date of birth"
+            required
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            I accsept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              required
+            >
+          </label>
+
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="field">
+
+          <label class="label">
+          E-mail:
+            <input type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            Password:
+            <input type="password"
+            required
+            minlength="5"
+            maxlength="10"
+            >
+          </label>
+
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting facts about you!</legend>
+
+        <div class="field">
+          Do you love cats?
+
+          <label class="label">
+            <input type="radio"
+            name="love cats"
+            value="yes"
+            >
+            Yes
+          </label>
+
+          <label class="label">
+            <input type="radio"
+            name="love cats"
+            value="no"
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            What is your favorite color?
+            <input type="color"
+            name="color"
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            What time do you go to bad?
+            <input type="time"
+            name="sleeptime"
+            >
+          </label>
+
+        </div>
+
+        <div class="field">
+
+          <label class="label">
+            What are your favorite brand of cars?
+
+            <select name="recommendation" multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+
+          </label>
+
+        </div>
+
+
+        <div class="field">
+
+          <label class="label">
+            How do you rate your work?
+            <input type="range"
+            name="works rate"
+            value="0"
+            >
+          </label>
+
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <div class="field">
+
+          <label class="label">
+            Comments:
+            <textarea name="comments"
+                      cols="20"
+                      rows="2">
+            </textarea>
```

Comment:
[checklist p.8 ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.